examples/siro_sandbox/README.md

```diff
@@ -1,136 +1,61 @@
-    - [Example launch command](#example-launch-command)
-  - [AppState: pick\_throw\_vr](#appstate-pick_throw_vr)
-  - [AppState: free\_camera](#appstate-free_camera)
-    - [Example launch command](#example-launch-command-1)
-  - [AppState: socialnav](#appstate-socialnav)
-  - [AppState: tutorial](#appstate-tutorial)
-- [Controls](#controls)
-- [Workaround for poor runtime perf on slower machines](#workaround-for-poor-runtime-perf-on-slower-machines)
-- [Command-line Options](#command-line-options)
-  - [Hack to hide the skinned humanoid in the GUI viewport](#hack-to-hide-the-skinned-humanoid-in-the-gui-viewport)
-  - [Saving episode data](#saving-episode-data)
-  - [Debugging visual sensors](#debugging-visual-sensors)
-  - [Debugging simulator-rendering](#debugging-simulator-rendering)
-  - [GUI-controlled agents and free camera mode](#gui-controlled-agents-and-free-camera-mode)
-  - [First-person and third-person mode for GUI-controlled humanoid](#first-person-and-third-person-mode-for-gui-controlled-humanoid)
-  - [Can grasp/place area](#can-graspplace-area)
-  - [Disable episode end on collision](#disable-episode-end-on-collision)
-  - [Play episodes filter](#play-episodes-filter)
-  - [Saving episode data](#saving-episode-data-1)
-  - [Capturing Gfx-Replay Files](#capturing-gfx-replay-files)
-  - [Human-in-the-loop tutorial sequence](#human-in-the-loop-tutorial-sequence)
-  - [Testing BatchReplayRenderer](#testing-batchreplayrenderer)
-- [Known Issues](#known-issues)
-- [Sandbox Tool Architecture](#sandbox-tool-architecture)
-- [VR HITL Evaluation](#vr-hitl-evaluation)
-
+- [Human-in-the-loop (HITL) Framework](#human-in-the-loop-hitl-framework)
+  - [Installation](#installation)
+  - [Example HITL applications](#example-hitl-applications)
+  - [VR HITL applications](#vr-hitl-applications)
+  - [Workaround for poor runtime perf on slower machines](#workaround-for-poor-runtime-perf-on-slower-machines)
+  - [Command-line Options](#command-line-options)
+    - [Hack to hide the skinned humanoid in the GUI viewport](#hack-to-hide-the-skinned-humanoid-in-the-gui-viewport)
+    - [Saving episode data](#saving-episode-data)
+    - [Debugging visual sensors](#debugging-visual-sensors)
+    - [Debugging simulator-rendering](#debugging-simulator-rendering)
+    - [GUI-controlled agents and free camera mode](#gui-controlled-agents-and-free-camera-mode)
+    - [First-person and third-person mode for GUI-controlled humanoid](#first-person-and-third-person-mode-for-gui-controlled-humanoid)
+    - [Can grasp/place area](#can-graspplace-area)
+    - [Disable episode end on collision](#disable-episode-end-on-collision)
+    - [Play episodes filter](#play-episodes-filter)
+    - [Saving episode data](#saving-episode-data-1)
+    - [Capturing Gfx-Replay Files](#capturing-gfx-replay-files)
+    - [Testing BatchReplayRenderer](#testing-batchreplayrenderer)
+  - [Known Issues](#known-issues)
+  - [HITL Framework Architecture](#hitl-framework-architecture)
```

Comment:
Would it be possible to add a quick description on code organization too i.e., what are apps and what lives in a particular app vs. sandbox driver vs. hitl main?

Reply:
I'll add this in a follow-up PR. [Asana item](https://app.asana.com/0/1205829436198608/1206335176254383)